### ISSUE TITLE
PDF accessibility audit tool for VA 508 office

### ISSUE THREAD

**Issue by kevwalsh** [department-of-veterans-affairs/va.gov-cms#6980]
## User Story or Problem Statement

As a 508 office accessibility reviewer...

1. I want a repository/directory/tool with which i can run accessibility reviews for PDFs that have been uploaded to VA.gov
2. I want to know who has added them, and what Section owns them (eg a VAMC system, or other)
3. I want to know what pages these PDFs have been added to, so that i can understand the context 

## Acceptance Criteria
- [x] VA 508 office can access a list of all PDFs that have been added, via CMS, to VA.gov. 
- [x] Ideally this list exists as a view in CMS, but can also be exported by CSV
- [x] Stretch: shows which user uploaded them, and to what Section.
- [ ] Stretch: List shows where these PDFs appear.
- [x] Whatever stretch items do not get finished are added to a [followup issue](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/8014).

Added after the first pass

Seeing this in action helps me think through a few ACs
- [ ] I think the path for view should be /admin/content/media/audit/pdfs, and the one for urls should be /admin/content/media/audit/image-alt, ideally with /admin/content/media as a parent menu item.
- [ ] Direct access to the PDF URLs in audit
  - [ ] https://www.va.gov/files/YYYY-MM//[filename].pdf is listed as column beside the name. I believe YYYY-MM is date created but that probably needs to be double checked. 
  - [ ] https://prod.cms.va.gov/[path-to-pdf] is listed as column
- [ ] Exposed filter label should be "Name contains" not "Name"
- [ ] Section should be linked. (minor but quick)
- [ ] We've validated "number of uses" is helpful, or misleading. [See screenshot in comment on PR](https://github.com/department-of-veterans-affairs/va.gov-cms/pull/8013#pullrequestreview-882140044)

## Implementation steps
<optional>

## Design principles

Veteran-centered
- [ ] `Single source of truth`: Increase reliability and consistency of content on VA.gov by providing a single source of truth.
- [ ] `Accessible, plain language`: Provide guardrails and guidelines to ensure content quality.
- [ ] `Purposely structured content`: Ensure Content API can deliver content whose meaning matches its structure.
- [ ] `Content lifecycle governance`: Produce tools, processes and policies to maintain content quality throughout its lifecycle.

Editor-centered
- [ ] `Purpose-driven`: Create an opportunity to involve the editor community in VA’s mission and content strategy goals.
- [ ] `Efficient`: Remove distractions and create clear, straightforward paths to get the job done.
- [ ] `Approachable`: Offer friendly guidance over authoritative instruction.
- [ ] `Consistent`: Reduce user’s mental load by allowing them to fall back on pattern recognition to complete tasks.
- [ ] `Empowering`: Provide clear information to help editors make decisions about their work.

### CMS Team

Please leave only the team that will do this work selected. If you're not sure, it's fine to leave both selected.

- [ ] `Platform CMS Team`
- [x] `Sitewide CMS Team`

**Comment by msbtterswrth** [comment]
There are two modules that may help us build something: https://www.drupal.org/project/entity_usage or https://www.drupal.org/project/entity_media_usage. I think the first one is a little more featured than the second but I wanted to drop in both just in case.
